site/src/api/typesGenerated.ts

```diff
@@ -620,3 +620,27 @@ export type WorkspaceAgentStatus = "connected" | "connecting" | "disconnected"
 
 // From codersdk/workspacebuilds.go
 export type WorkspaceTransition = "delete" | "start" | "stop"
+
+// TODO: Remove this when the generated types work for AuditLogs
+export interface AuditLog {
```

Comment:
Will this get overwritten when types are regenerated? Maybe it should go in types.ts instead?

Reply:
It should be overwritten!